src/shared/constants/WordAttributes.js

```diff
@@ -11,6 +11,10 @@ export default {
     value: 'isCommon',
     label: 'Is Common',
   },
+  FREQUENCY: {
+    value: 'frequency',
+    label: 'Frequency',
+  },
```

Comment:
this is close, but we probably wouldn't want frequency in the `attributes` portion of the word document since attributes are typically boolean values - we can remove this change in this file.

we already have a `frequency` field on the Word model that you can use instead